stash_engine/lib/stash/download/version.rb

```diff
@@ -35,6 +39,14 @@ def download(resource:)
         end
       end
 
+      # make it give a filename based on DOI and version with stuff sanitized out
+      def disposition_filename
```

Comment:
I feel like this should somehow be consolidated with the sanitization happening in `app/models/stash_engine/file_upload.rb`.

Reply:
I agree with you about this.  I wasn't sure what to do since it does something similar, but not exactly the same.

I didn't really want to strip out characters that were already in there and we support.

I honestly feel most conflicted about spaces.  I know Ryan hates them, but they are so widely supported by most all OSes and other things that I think it's weird not to support them.  If we're going to change them, then I think going in is the place to do it as some kind of "best practice filenames policy" rather than it being any kind of technical limitation.